spec/acceptance/nodesets/default.yml

```diff
@@ -14,7 +14,7 @@
   end
 -%>
 HOSTS:
-  puppet:
+  puppet.test:
```

Comment:
This change broke other portions of the test

Reply:
Also, the ipa suite nodeset was not similarly adjusted.  Need to be consistent, especially, since the ipa and simp_lite suites use some of the same some shared_examples and helpers used by the default suite.  This also means that the ipa and simp_lite suites need to pass in GitLab before this PR can be merged.